src/marketplace-custom-config.js

```diff
@@ -1,77 +1,150 @@
 /*
  * Marketplace specific configuration.
+ *
+ * Every filter needs to have following keys:
+ * - **id**: unique id of the filter
+ * - **label**: The default label of the filter
+ * - **type**: string that represents one of the existing filter components
```

Comment:
You could either list the possible values here, or mention that they can be seen in `FilterComponent.js`.